pages/common/sponge.md

```diff
@@ -0,0 +1,12 @@
+# sponge
+
+> Soak up standard input and write to a file.
+> Sponge soaks up all its input before writing the output file.
+
+- Appended all file content to its origin file:
+
+`cat {{path/to/file}} | sponge -a {{/path/to/file}}`
+
+- Remove all line start with # in file:
+
+`grep -v '^#' {{path/to/file}} | sponge {{/path/to/file}}`
```

Comment:
```suggestion
`grep -v '^{{#}}' {{path/to/file}} | sponge {{/path/to/file}}`
```
Is `path/to/file` in `sponge` supposed to be output file? I guess that should be changed / clarified somehow.